onnxruntime/core/providers/cpu/tensor/upsamplebase.h

```diff
@@ -41,7 +41,8 @@ enum ResizeCoordinateTransformationMode {
   TF_HALF_PIXEL_FOR_NN = 3,
   ALIGN_CORNERS = 4,
   TF_CROP_AND_RESIZE = 5,
-  CoordinateTransformationModeCount = 6,
+  HALF_PIXEL_SYMMETRIC = 6,
+  CoordinateTransformationModeCount = 7,
```

Comment:
Out of curiosity: wouldn't it be just a bit helpful to retain the old value of 6 for CoordinateTransformationModeCount and make HALF_PIXEL_SYMMETRIC  7? May be this is nit, but seems useful to me.